### PR TITLE
[Chore] Update README

### PR DESCRIPTION
**Pull request by hoangnguyen92dn**
## What happened 👀

Update project README to provide guideline for internal and external developers who want to use/contribute to the project

## Insight 📝

Update README and add LICENSE file

## Proof Of Work 📹

N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,56 @@
...
+    - Kotlin DSL
+
+    ```kotlin
+    implementation("com.github.nimblehq:android-common-ktx:<version>")
+    ```
+
+## Contributing
 
-`git clone git@github.com:nimblehq/android-common-ktx.git`
+For major changes, please open an issue first to discuss what you would like to change. Visit our [Wiki](https://github.com/nimblehq/android-common-ktx/wiki#how-to-contribute) for guidelines to contribute your ideas to our project.
```
Comment (lydiasama) [inline README.md:53]:
```suggestion
The contribution is welcome by opening an issue for discussion. You can visit our [Wiki](https://github.com/nimblehq/android-common-ktx/wiki#how-to-contribute) for guidelines to contribute your ideas to our project.
```
What do you think? @hoangnguyen92dn

**Review by lydiasama — COMMENTED** [review]
The rest LGTM. 👍🏻

File: README.md (hunk 1 above)
Comment (hoangnguyen92dn) [reply README.md:53]: @lydiasama Nice 👍

File: README.md [hunk 2]
```diff
@@ -1,12 +1,56 @@
...
+    - Kotlin DSL
+
+    ```kotlin
+    repositories {
+    	...
+    	maven(url = "https://jitpack.io")
+    }
+    ```
+
+2. Add the dependency in the application build.gradle
```
Comment (doannimble) [inline README.md:36]: @hoangnguyen92dn Should we use `build.gradle` ?

File: README.md [hunk 3]
```diff
@@ -1,12 +1,56 @@
...
 # Android Common Extensions
 
 This repository provides a collection of common Android extensions.
 
+[![](https://jitpack.io/v/nimblehq/android-common-ktx.svg)](https://jitpack.io/#nimblehq/android-common-ktx)
+
 ## Usage
 
-Clone the repository
+1. Add JitPack to your project build.gradle
```
Comment (doannimble) [inline README.md:15]: @hoangnguyen92dn Suggestion:

```
Add JitPack to your project `build.gradle` or `settings.gradle`
```

**Comment by doannimble** [comment]
Rest lgtm

File: README.md (hunk 3 above)
Comment (hoangnguyen92dn) [reply README.md:15]: @doannimble It should be `or` or `and`? 🤔

**Review by lydiasama — APPROVED** [review]

**Review by doannimble — APPROVED** [review]
Good to go 🏃‍♂️

**Pull request merged**